contracts/Super1155.sol

```diff
@@ -1031,6 +995,42 @@ contract Super1155 is PermitControl, ERC165Storage, IERC1155, IERC1155MetadataUR
     emit TransferBatch(operator, _burner, address(0), _ids, _amounts);
   }
 
+  /**
+    This function allows an address to destroy some of its items.
+
+    @param _burner The address whose item is burning.
+    @param _id The item ID to burn.
+    @param _amount The amount of the corresponding item ID to burn.
+  */
+  function burn(address _burner, uint256 _id, uint256 _amount) external virtual{
+      require(_hasItemRight(_id, BURN), "Super1155: you don't have rights to burn");
+      burnBatch(_burner, _asSingletonArray(_id), _asSingletonArray(_amount));
+    /*
+    // Retrieve the group ID from the given item `_id` and check burn validity.
```

Comment:
If this works let's delete the dead comments.

Reply:
Ok